client/src/locales/fr-FR.json

```diff
@@ -106,6 +106,7 @@
             "forgottenPassword": "Mot de passe oublié ?",
             "emailLabel": "E-mail",
             "passwordLabel": "Mot de passe",
+            "passwordError": "Le mot de passe est incorrect",
```

Comment:
English text ends with a `.`, French text does not.

Reply:
Done